### DISCUSSION TITLE
Library (Layerchart) classes not interpreted

### DISCUSSION THREAD

**Discussion by mvuidev** [unocss/unocss#4320] [Q&A]
Hi there,

I try to use LayerChart library on a Sveltekit + UnoCSS project.
My problem is that classes from LayerChart are not interpreted.
I added the path in my uno.config.js file but it does not help :

```js
content: {
	filesystem: ['./node_modules/layerchart/**/*.{svelte,js}'],
	pipeline: {
		include: [
			'./node_modules/layerchart/**/*.{svelte,js}'
		]
	}
},
```

Am I missing something ?

Edit : I use @unocss/svelte-scoped/vite and this is the problem. If I use unocss/vite, it's OK.

Any way to make Uno interpret classes of libraries using svelte-scoped ?

**Comment by jamesarosen** [comment]
I have the same experience when using both `@unocss/webpack` and `@unocss/postcss`.

The only solution I've found is to use `@unocss/cli` and compile the `uno.css` file out-of-band from my main development process.
